cmd/gcs.go

```diff
@@ -0,0 +1,101 @@
+package cmd
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/pingcap-inc/tidb2dw/pkg/apiservice"
+	"github.com/pingcap-inc/tidb2dw/pkg/tidbsql"
+	"github.com/pingcap/errors"
+	"github.com/pingcap/log"
+	"github.com/pingcap/tiflow/pkg/logutil"
+	"github.com/spf13/cobra"
+	"github.com/thediveo/enumflag"
+	"go.uber.org/zap"
+)
+
+func NewGCSCmd() *cobra.Command {
+	var (
+		tidbConfigFromCli   tidbsql.TiDBConfig
+		credentialsFilePath string
+		tables              []string
+		snapshotConcurrency int
+		storagePath         string
+		cdcHost             string
+		cdcPort             int
+		cdcFlushInterval    time.Duration
+		cdcFileSize         int64
+		logFile             string
+		logLevel            string
+
+		mode          RunMode
+		apiListenHost string
+		apiListenPort int
+	)
+
+	run := func() error {
+		err := logutil.InitLogger(&logutil.Config{
+			Level: logLevel,
+			File:  logFile,
+		})
+		if err != nil {
+			return errors.Trace(err)
+		}
+
+		storageURI, err := getGCSURIWithCredentials(storagePath, credentialsFilePath)
+		if err != nil {
+			return errors.Trace(err)
+		}
+
+		snapshotURI, incrementURI, err := genSnapshotAndIncrementURIs(storageURI)
+		if err != nil {
+			return errors.Trace(err)
+		}
+
+		_, err = Export(
+			&tidbConfigFromCli, tables, storageURI, snapshotURI, incrementURI, snapshotConcurrency,
+			cdcHost, cdcPort, cdcFlushInterval, cdcFileSize, mode)
+
+		return err
+	}
+
+	cmd := &cobra.Command{
+		Use:   "gcs",
+		Short: "Export snapshot and incremental data from TiDB to GCS",
+		Run: func(_ *cobra.Command, _ []string) {
+			runWithServer(mode == RunModeCloud, fmt.Sprintf("%s:%d", apiListenHost, apiListenPort), func() {
```

Comment:
For `export only`, it should not have `cloud` mode?

Reply:
You're right, it doesn't have some long-run progress.